Cargo.toml

```diff
@@ -91,7 +91,7 @@ toml = "0.8.20"
 rpassword = "7.3.1"
 promptly = "0.3.1"
 ptree = "0.5.2"
-reqwest = "0.12.14"
+reqwest = "0.12.15"
```

Comment:
Is this needed?

Reply:
I think I needed to bump because some of the dependencies upgraded theirs and it was conflicting